Lib/ufo2ft/otfPostProcessor.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function, division, absolute_import, unicode_literals
+
+import tempfile
+
+from fontTools.ttLib import TTFont
+
+
+class OTFPostProcessor(object):
+    """Does some post-processing operations on a compiled OpenType font, using
+    info from the source UFO where necessary.
+    """
+
+    def __init__(self, otf, ufo):
+        self.ufo = ufo
+        tmp_file = tempfile.NamedTemporaryFile()
+        tmp_path = tmp_file.name
+        otf.save(tmp_path)
+        self.otf = TTFont(tmp_path)
+
+    def process(self, useProductionNames=True):
+        if useProductionNames:
+            self._rename_glyphs_from_ufo()
+        return self.otf
+
+    def _rename_glyphs_from_ufo(self):
```

Comment:
Can you combine all of these together and put it in Snippets please?  Otherwise we can do it together next week.  (travelling this week and busy with other stuff.)